### PR TITLE
CLOUDP-61290: Validate base url

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ [CLOUDP-61290](https://jira.mongodb.org/browse/CLOUDP-61290)

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

This is a breaking change, users will need to update their base url to include a trailing slash

### CODE REVIEW

File: internal/cli/cli.go [hunk 1]
```diff
@@ -37,14 +37,6 @@ type globalOpts struct {
 	projectID string
 }
 
-func validateObjectID(s string) error {
```
Comment (gssbzn) [inline internal/cli/cli.go:40]: moved to its own package

**Comment by andreaangiolillo** [comment]
something seems broken in the e2e test. Do you think it is related to this changeset?

**Comment by gssbzn** [comment]
@andreaangiolillo yes I need to change a setting in drone but since you have a PR I'm waiting for that as the change would break you instead

**Comment by gssbzn** [comment]
@andreaangiolillo fix

**Pull request merged**